commands/mining_daemon_test.go

```diff
@@ -81,6 +76,15 @@ func TestMiningSealNow(t *testing.T) {
 	miningAddress, err := minerNode.MiningAddress(ctx)
 	require.NoError(t, err)
 
+	// start mining for miner node to seal and schedule PoSting
+	require.NoError(t, minerNode.MiningStart(ctx))
+	defer func() {
+		require.NoError(t, minerNode.MiningStop(ctx))
+	}()
+
+	// Mine the commitSector and the submitPoSt
+	series.CtxMiningNext(ctx, 2)
```

Comment:
Since this test still uses `MiningStart`, this is confusing at best.

I *think* what's happening is that the genesis node is not mining, and this `MiningNext` is an instruction to it. But the node under test `minderNode` is mining. But it has no power yet and so won't produce blocks? If that's write, please describe this situation in a comment.

Later, I think we want to separate block production from storage-power-mining (and both from deal-making), after which this could be more clear, and we could prevent the node under test from attempting to produce blocks. Re #3179, #3140